schedule-replicas-by-topology-labels.md

```diff
@@ -16,6 +16,12 @@ To make this mechanism effective, you need to properly configure TiKV and PD so
 
 ## Configure `labels` based on the cluster topology
 
+> **Note:**
+>
+> Location labels are defined as Strings in PD with no restricted default or pre-configured values. This offers customizable label configuration in label names other than `zone`, `rack` or `host` which are used as demonstration in below sections.
```

Comment:
```suggestion
> Location labels are defined as strings in PD with no restricted default or pre-configured values. This offers customizable label configuration in label names other than `zone`, `rack` or `host` which are used as demonstration in below sections.
```

Reply:
should "Location labels" be changed to `location-labels`? Also [here](https://github.com/pingcap/docs/blob/7ba53b86478e443ddd76e5e88df8ff3bbc2e6f68/schedule-replicas-by-topology-labels.md?plain=1#L63) has already mentioned the similar concept in the same doc.